CONTRIBUTING.md

```diff
@@ -1,3 +1,153 @@
 # Getting Involved
 
-If you have any experience with writing CLIs, Docker or Sails and Ember itself and you are interested in seeing this happening, just get in touch or open a new issue here.
\ No newline at end of file
+If you have any experience with writing CLIs, Docker, or with Sails and/or Ember
+themselves and you are interested in seeing this happening, just get in touch
+on our [Gitter chat](https://gitter.im/artificialio/sane) or open a new issue here.
+
+# Questions
+
+This is the repository and issue tracker for SANE. The community uses this
+site to collect and track bugs and discussions of new features. If you are
+having difficulties using SANE or have a question about usage
+please ask a question in the Gitter chat: https://gitter.im/artificialio/sane
+
+# Issues
+
+Think you've found a bug or have a new feature to suggest? Let us know!
+
+## Reporting a Bug
+
+1. Update to the most recent master release if possible. We may have already
+   fixed your bug.
+
+2. Search for similar issues. It's possible somebody has encountered this bug
+   already.
+
+3. Provide a demo that specifically shows the problem. This demo should be fully
+   operational with the exception of the bug you want to demonstrate. The more
+   pared down, the better. Issues with demos are prioritized.
+
+4. Your issue will be verified. The provided demo will be tested for
+   correctness. The SANE team will work with you until your issue can be
+   verified.
+
+5. Keep up to date with feedback from the SANE team on your ticket. Your
+   ticket may be closed if it becomes stale.
+
+6. If possible, submit a Pull Request with a failing test. Better yet, take
+   a stab at fixing the bug yourself if you can!
+
+The more information you provide, the easier it is for us to validate that
+there is a bug and the faster we'll be able to take action.
+
+## Requesting a Feature
+
+1. Search Issues for similar feature requests. It's possible somebody has
+   already asked for this feature or provided a pull request that we're still
+   discussing.
+
+2. Provide a clear and detailed explanation of the feature you want and why it's
+   important to add. Keep in mind that we want features that will be useful to
+   the majority of our users and not just a small subset. If you're just
+   targeting a minority of users, consider writing an addon library for
+   SANE.
+
+3. If the feature is complex, consider writing some initial documentation for
+   it. If we do end up accepting the feature it will need to be documented and
+   this will also help us to understand it better ourselves.
```

Comment:
Ah, should have quoted better: "public documentation has a genuine influence on the resulting API." That was, what I read.

So the idea was to mention something along these lines: "If you provide a Feature Request with documentation this will help shaping the implemented API".

Does that make it more clear? But not sure if that is actually adding anything.